src/pages/[locale]/demo/native-features/example-with-ssg-and-revalidate.tsx

```diff
@@ -13,14 +13,15 @@ import {
 } from '@/layouts/demo/demoSSG';
 import { Product } from '@/modules/core/data/types/Product';
 import timeDifference from '@/modules/core/date/timeDifference';
-import createApolloClient from '@/modules/core/gql/graphql';
-import withApollo from '@/modules/core/gql/hocs/withApollo';
+import { initializeApollo } from '@/modules/core/apollo/apolloClient';
```

Comment:
Maybe keeping the name `createApolloClient` would make more sense? 🤔 
Rather than `initializeApollo`, which is not always true since it uses a memo/cached version internally and doesn't always initialize it.

Reply:
Let's keep it that way.